### PR TITLE
musescore: update to 3.5.2.

### PR DESCRIPTION
**Pull request by tibequadorian**

### CODE REVIEW

**Comment by ericonr** [comment]
The build shows `Version 0.0 of the MuseScore SoundFont is outdated, downloading version 0.2.0.`, would it be possible to instead add it as a distfile?

**Comment by tibequadorian** [comment]
in fact it downloads not one but 5 files from https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General/
MuseScore_General.sf3
MuseScore_General_Changelog.md
MuseScore_General_License.md
MuseScore_General_Readme.md
VERSION

although I think Changelog and Readme are not that important. It's not impossible but worth it? I think it overcomplicates the template.

**Comment by ericonr** [comment]
Are those downloads checksummed, at least?

**Comment by tibequadorian** [comment]
> Are those downloads checksummed, at least?
i dont think so
https://github.com/musescore/MuseScore/blob/465e7b6fe50973a24a75891e27575a8d2b962f6a/CMakeLists.txt#L337-L379

**Comment by ericonr** [comment]
Could we try to package the fonts, instead? I'd strongly prefer to have these in distfiles or as a separate package.

**Comment by tibequadorian** [comment]
Maybe it's a job for the maintainer then... I just wanted to update the package

**Comment by ericonr** [comment]
@tibequadorian can you leave a `# TODO:` comment in the template about this, please? Then I can merge the update.

**Comment by tibequadorian** [comment]
like this?

File: srcpkgs/musescore/template [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Template file for 'musescore'
+# TODO: Add MuseScore Soundfont as distfiles or separate package instead of
+#       downloading in the build process.
+#       https://ftp.osuosl.org/pub/musescore/soundfont/MuseScore_General/
```
Comment (ericonr) [inline srcpkgs/musescore/template:4]: Doesn't need the indentation, but otherwise it's great. Thanks!

**Comment by ericonr** [comment]
Thank you!

**Pull request merged**